spec/registry.xml

```diff
@@ -0,0 +1,250 @@
+      <!-- The unit to use in unit formatting.
+        Possible values are core unit identifiers, defined in UTS #35, Part 2, Section 6.
+        A subset of units from the full list was selected for use in ECMAScript.
+        Pairs of simple units can be concatenated with "-per-" to make a compound unit.
+        There is no default value; if the style is "unit", the unit property must be provided.
+      -->
+      <option name="unit" pattern="anythingNotEmpty"/>
+      <!-- The unit formatting style to use in unit formatting. -->
+      <option name="unitDisplay" values="long short narrow" default="short"/>
+      <!-- The minimum number of integer digits to use.
+        A value with a smaller number of integer digits than this number will be
+        left-padded with zeros (to the specified length) when formatted.
+      -->
+      <option name="minimumIntegerDigits" values="1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21" default="1"/>
+      <!-- The minimum number of fraction digits to use.
+        The default for plain number and percent formatting is 0;
+        the default for currency formatting is the number of minor unit digits provided by
+        the ISO 4217 currency code list (2 if the list doesn't provide that information).
+      -->
+      <option name="minimumFractionDigits" values="0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20"/>
+      <!-- The maximum number of fraction digits to use.
+        The default for plain number formatting is the larger of minimumFractionDigits and 3;
+        the default for currency formatting is the larger of minimumFractionDigits and the number of minor
+        unit digits provided by the ISO 4217 currency code list (2 if the list doesn't provide that information);
+        the default for percent formatting is the larger of minimumFractionDigits and 0.
+      -->
+      <option name="maximumFractionDigits" values="0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20"/>
+      <!-- The minimum number of significant digits to use. -->
+      <option name="minimumSignificantDigits" values="1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21" default="1"/>
+      <!-- The maximum number of significant digits to use. -->
+      <option name="maximumSignificantDigits" values="1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21" default="21"/>
+
+    </formatSignature>
+
+    <!-- TODO: IGNORED for now are the "Experimental" options. To discuss?
+      signDisplay@negative, useGrouping, roundingMode, roundingPriority, roundingIncrement, trailingZeroDisplay
+    -->
+
+  </function>
+
+  <function name="plural">
+    <!-- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/PluralRules/PluralRules
+      https://unicode-org.github.io/icu-docs/apidoc/dev/icu4j/com/ibm/icu/text/MessageFormat.html
+    -->
+    <description>Language-sensitive plural selection</description>
+    <matchSignature>
```

Comment:
My preference would be to include this `<matchSignature>` under `<function name="plural">`. Given that this is likely to be a lively topic of discussion, it might be easier to separate it out to a separate PR.

Reply:
?

This `<matchSignature>` (line 186) is under `<function name="plural">` (line 181).
We just have a a comment and `<description>` in between.
And `description` is mandatory to be present, and mandatory to be before `matchSignature` (per DTD). and

Reply:
Ach, sorry, mistyped earlier. To clarify:

My preference would be to include this `<matchSignature>` under `<function name="number">`. Given that this is likely to be a lively topic of discussion, it might be easier to separate it out to a separate PR.

Reply:
> My preference would be to include this <matchSignature> under <function name="number">. 

I think this is covered by the issue you opened (https://github.com/unicode-org/message-format-wg/issues/425). Thanks.